fission/src/Synthesis.tsx

```diff
@@ -182,46 +183,48 @@ function Synthesis() {
-    <ImportLocalMirabufModal key="import-local-mirabuf" modalId="import-local-mirabuf" />,
+    <SettingsModal modalId="settings" />,
+    <SpawningModal modalId="spawning" />,
+    <AddRobotsModal modalId="add-robot" />,
+    <AddFieldsModal modalId="add-field" />,
+    <ViewModal modalId="view" />,
+    <DownloadAssetsModal modalId="download-assets" />,
+    <RoboRIOModal modalId="roborio" />,
+    <DrivetrainModal modalId="drivetrain" />,
+    <ThemeEditorModal modalId="theme-editor" />,
+    <ExitSynthesisModal modalId="exit-synthesis" />,
+    <MatchResultsModal modalId="match-results" />,
+    <UpdateAvailableModal modalId="update-available" />,
+    <ConnectToMultiplayerModal modalId="connect-to-multiplayer" />,
+    <ServerHostingModal modalId="server-hosting" />,
+    <ChangeInputsModal modalId="change-inputs" />,
+    <ResetAllInputsModal modalId="reset-inputs" />,
+    <ChooseMultiplayerModeModal modalId="multiplayer-mode" />,
+    <ChooseSingleplayerModeModal modalId="singleplayer-mode" />,
+    <PracticeSettingsModal modalId="practice-settings" />,
+    <DeleteThemeModal modalId="delete-theme" />,
+    <DeleteAllThemesModal modalId="delete-all-themes" />,
+    <NewThemeModal modalId="new-theme" />,
+    <RCCreateDeviceModal modalId="create-device" />,
+    <RCConfigPwmGroupModal modalId="config-pwm" />,
+    <RCConfigEncoderModal modalId="config-encoder" />,
+    <MatchModeModal modalId="match-mode" />,
+    <SpawningModal modalId="spawning" />,
+    <ConfigMotorModal modalId="config-motor" />,
+    <ManageAssembliesModal modalId="manage-assemblies" />,
+    <ImportLocalMirabufModal modalId="import-local-mirabuf" />,
 ]
 
 const initialPanels: ReactElement[] = [
-    <RobotSwitchPanel key="multibot" panelId="multibot" openLocation="right" sidePadding={8} />,
-    <DriverStationPanel key="driver-station" panelId="driver-station" />,
-    <SpawnLocationsPanel key="spawn-locations" panelId="spawn-locations" />,
-    <ScoreboardPanel key="scoreboard" panelId="scoreboard" />,
-    <ConfigureGamepiecePickupPanel key="config-gamepiece-pickup" panelId="config-gamepiece-pickup" />,
-    <ConfigureShotTrajectoryPanel key="config-shot-trajectory" panelId="config-shot-trajectory" />,
-    <ScoringZonesPanel key="scoring-zones" panelId="scoring-zones" />,
-    <ZoneConfigPanel key="zone-config" panelId="zone-config" />,
+    <RobotSwitchPanel panelId="multibot" openLocation="right" sidePadding={8} />,
+    <DriverStationPanel panelId="driver-station" />,
+    <SpawnLocationsPanel panelId="spawn-locations" />,
+    <ScoreboardPanel panelId="scoreboard" />,
+    <ConfigureGamepiecePickupPanel panelId="config-gamepiece-pickup" />,
+    <ConfigureShotTrajectoryPanel panelId="config-shot-trajectory" />,
+    <ScoringZonesPanel panelId="scoring-zones" />,
+    <ZoneConfigPanel panelId="zone-config" />,
+    <ImportMirabufPanel panelId="import-mirabuf" />,
     <PokerPanel key="poker" panelId="poker" />,
 ]
```

Comment:
We recently added keys to all the modals and panels I thought. Are we removing them already?

Reply:
UUUUGGGHHHHH gimme a bit

Reply:
How did I not see that

Reply:
Just fixed it